### PR TITLE
fix: add missing env variables, and move them into deployments

### PR DESCRIPTION
**Pull request by barnabasbusa**

### CODE REVIEW

File: charts/blobscan-api/templates/deployment.yaml [hunk 1]
```diff
@@ -74,6 +74,16 @@ spec:
...
+            - name: BLOBSCAN_API_PORT
+              value: {{ .Values.httpPort | quote }}
+            - name: DATABASE_URL
+              value: {{ .Values.config.DATABASE_URL | quote }}
+            - name: SECRET_KEY
+              value: {{ .Values.config.SECRET_KEY | quote }}
+            - name: BEACON_NODE_ENDPOINT
+              value: {{ .Values.config.BEACON_NODE_ENDPOINT | quote }}
+            - name: CHAIN_ID
+              value: {{ .Values.CHAIN_ID | quote }}
```
Comment (PabloCastellano) [inline charts/blobscan-api/templates/deployment.yaml]: Shouldn't be `.Values.config.CHAIN_ID`?

File: charts/blobscan/values.yaml [hunk 2]
```diff
@@ -24,16 +24,19 @@ httpPort: 3000
...
 args: []
 
+# -- Application configuration
+config:
+  NEXTAUTH_URL: ""
+  DATABASE_URL: "postgresql://postgres:postgres@blobscan-blobscandb:5432/blobscan?schema=public"
+  SECRET_KEY: "supersecret"
+  NEXT_PUBLIC_NETWORK_NAME: "mainnet"
+  NEXT_PUBLIC_BEACON_BASE_URL: ""
+  NEXT_PUBLIC_EXPLORER_BASE_URL: ""
```
Comment (PabloCastellano) [inline charts/blobscan/values.yaml]: Let's keep the sane defaults so it works out of the box until we configure these variables as optional

File: charts/blobscan/README.md [hunk 3]
```diff
@@ -36,11 +36,12 @@ Blobscan is the first blockchain explorer that helps to navigate and visualize t
...
+| config | object | `{"BEACON_NODE_ENDPOINT":"http://beacon-node:5052","DATABASE_URL":"postgresql://postgres:postgres@blobscan-blobscandb:5432/blobscan?schema=public","NEXTAUTH_URL":"","NEXT_PUBLIC_BEACON_BASE_URL":"","NEXT_PUBLIC_EXPLORER_BASE_URL":"","NEXT_PUBLIC_NETWORK_NAME":"mainnet","NODE_ENV":"","SECRET_KEY":"supersecret"}` | Application configuration |
 | containerSecurityContext | object | See `values.yaml` | The security context for containers |
 | customArgs | list | `["web"]` | Custom args for the blobscan container |
 | customCommand | list | `[]` | Command replacement for the blobscan container |
 | extraContainers | list | `[]` | Additional containers |
-| extraEnv | list | `[{"name":"DATABASE_URL","value":"postgresql://postgres:postgres@blobscan-blobscandb:5432/blobscan?schema=public"},{"name":"SECRET_KEY","value":"supersecret"},{"name":"NEXT_PUBLIC_BEACON_BASE_URL","value":"http://134.209.87.158:8080/"},{"name":"NEXT_PUBLIC_EXPLORER_BASE_URL","value":"https://explorer.4844-devnet-7.ethpandaops.io/"}]` | Additional env variables |
+| extraEnv | list | `[]` | Additional env variables |
 | extraPodPorts | list | `[]` | Extra Pod ports |
```
Comment (PabloCastellano) [inline charts/blobscan/README.md:45]: Same, let's keep the defaults

**Review by PabloCastellano — COMMENTED** [review]
Thanks!

File: charts/blobscan-api/templates/deployment.yaml (hunk 1 above)
Comment (barnabasbusa) [reply charts/blobscan-api/templates/deployment.yaml]: Yes, you are right. Fixed.

File: charts/blobscan/values.yaml (hunk 2 above)
Comment (barnabasbusa) [reply charts/blobscan/values.yaml]: Set them to sane defaults.

File: charts/blobscan/README.md (hunk 3 above)
Comment (barnabasbusa) [reply charts/blobscan/README.md:45]: fixed

**Comment by PabloCastellano** [comment]
Thanks a lot! I will continue the work from here.

**Pull request merged**